cleanco/classify.py

```diff
@@ -3,59 +3,69 @@
 
 Examples of use:
 
->> # check name for its possible business type(s)
->> classification_sources = typesources()
->> matches("MyCompany Ltd", classification_sources)
-['Limited']
->>
-
->> # check name for its possible jurisdictions, usually countries
->> classification_sources = countrysources()
->> matches("MyCompany Ltd", classification_sources)
-['New Zealand', 'United Kingdom', 'United States of America']
->>
+# check name for its possible business type(s)
+>> matches("MyCompany Ltd", "local_name")
+['Private Limited by Guarantee', 'Private Limited Company', 'Limited Liability Company', 'New Generation Co-operatives']
+
+# check name for its possible countries
+>> matches("MyCompany Ltd", "country")
+['United States of America', 'United Kingdom', 'Canada']
 
 """
 
-from .termdata import terms_by_country, terms_by_type
+from iso20275 import Elf, ElfEntry
 from .clean import strip_tail, normalized
 
 
-def typesources():
-   "business types / abbreviations sorted by length of business type"
-   types = []
-   for business_type in terms_by_type:
-       for item in terms_by_type[business_type]:
-           types.append((business_type, item))
+def classifierList():
+    property_names = [p for p in dir(ElfEntry) if isinstance(
+        getattr(ElfEntry, p), property)]
+    return(property_names)
 
-   return sorted(types, key=lambda part: len(part[1]), reverse=True)
 
+def idClassification(abbreviation, source):
```

Comment:
This function is of key importance that would often be directly used. Would it therefore deserve a docstring? Is the name descriptive of what the function does? By the way I think cleanco mostly now uses lowercase identifiers rather than mixedCase?

Reply:
I renamed it to match the rest of the code.